cobigen-templates/templates-devon4j/src/main/templates/crud_java_server_app_complex/templates/java/${variables.rootPackage}/${variables.component}/logic/impl/${variables.component#cap_first}Impl.java.ftl

```diff
@@ -14,6 +14,17 @@ import java.util.List;
 import javax.inject.Inject;
 import javax.inject.Named;
 
+<#assign compositeIdVar = false>
+<#list pojo.fields as field>
+	<#if field.type?starts_with("Composite")>
+		<#assign compositeIdVar = true>
+		<#assign compositeIdTypeVar = field.type>
+	</#if>
+</#list>
```

Comment:
missing refactoring

Reply:
Changes done and committed. Re-Review required.